courses/tasks_test.py

```diff
@@ -1 +1,19 @@
-"""Course tasks tests"""
+"""Tests for Course related tasks"""
+
+import pytest
+
+from courses.tasks import sync_courseruns_data
+from courses.factories import CourseRunFactory
+
+pytestmark = [pytest.mark.django_db]
+
+
+def test_sync_courseruns_data(mocker):
+    """Test sync_courseruns_data calls the right api functionality from courses"""
+    sync_course_runs = mocker.patch("courses.utils.sync_course_runs")
```

Comment:
If the inline import above was to pass the test, Could you try without making it inline and instead mocking it like:

``` sync_course_runs = mocker.patch("courses.tasks.sync_course_runs")
```